bundles/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/handler/SerialBridgeThingHandler.java

```diff
@@ -94,6 +94,11 @@ public void initializeLater() {
             return;
         }
 
+        try {
+            Thread.sleep(5500); // add small delay for serviceloader to be ready
```

Comment:
Is there a better way? waiting for 5,5 seconds like this is somewhat error prone. No hook ? or loop with smaller delay and ready check?

Reply:
done